www/shared/styles/imports/mixins/layout-main.css

```diff
@@ -0,0 +1,26 @@
+@import "../variables";
+@import "../custom-medias";
+
+/* Usage: @mixin layout-main .content-selector;;
+============================================= */
```

Comment:
This divider seems a bit lost here, no? 🤔

Reply:
I don't think so, but there is an extra `;` at the end.

Reply:
I removed the extra `;`